docs/user-guide/system-admin/configuration/infinispan.md

```diff
@@ -12,14 +15,38 @@ The Infinispan configuration will be located inside the following module in `sta
 
 Keep in mind that the module version may vary depending on your JBoss version.
 
-### Configuration for Internal data caches
+### Hibernate Cache
+
+The following is the recommended configuration for the Hibernate cache:
+
+```xml
+...
+<cache-container name="hibernate" default-cache="local-query" jndi-name="java:jboss/infinispan/container/hibernate" start="EAGER" module="org.jboss.as.jpa.hibernate:4">
```

Comment:
Well, it does work out of the box, but you have to tell it that a) you want it to use infinispan as the cache provider (i.e. use the InifnispanRegionFactory), and 2) where is the inifinispan cache you want to use (i.e. jndi)